lib/systems/default.nix

```diff
@@ -91,7 +91,7 @@ let
         && final.parsed.kernel == platform.parsed.kernel;
       isCompatible = _: throw "2022-05-23: isCompatible has been removed in favor of canExecute, refer to the 22.11 changelog for details";
       # Derived meta-data
-      useLLVM = final.isFreeBSD || final.isOpenBSD;
+      useLLVM = final.isFreeBSD || final.isOpenBSD || final.isLLVMLibc;
```

Comment:
OOC does it make sense to ever use LLVM libc with GCC?

Reply:
Hard to say based on whether GCC supports the naked attribute for specific targets. I was thinking the LLVM abi which means to use the libc that it'd make sense to use the entire toolchain. That's what the team upstream thinks as well.